raiden/raiden_service.py

```diff
@@ -864,15 +863,13 @@ def handle_state_changes(self, state_changes: List[StateChange]) -> List[Greenle
                 raiden_events.extend(events)
 
         return self._trigger_state_change_effects(
-            old_state=old_state,
             new_state=views.state_from_raiden(self),
             state_changes=state_changes,
             events=raiden_events,
         )
 
     def _trigger_state_change_effects(
         self,
-        old_state: ChainState,
```

Comment:
I would be hesitant to remove this from the interface, but I couldn't find another reason why it was there in the first place, since it was introduced with the call to `update_monitoring_service_from_balance_proof( )`.
Also, it's a private method, so looks good.